### PR TITLE
Improved some things to the Migration tutorials and added directory where the two paper configs are

### PR DESCRIPTION
**Pull request by PetarMc1**
Improved Migration From CraftBukkit or Spigot  by adding another step
Improved Migration From Vanilla  by adding another step
The step that I added to both tutorials: 
Rename the downloaded file to match the name specified in the start command

### CODE REVIEW

File: docs/paper/admin/getting-started/migration.md [hunk 1]
```diff
@@ -22,17 +22,18 @@ It's easy to migrate from CraftBukkit or Spigot to Paper. Follow the steps below
 
 1. Stop your server if it is running, and create a full backup.
 2. Download Paper from [our downloads page](https://papermc.io/downloads).
-3. Replace your existing server jar with your freshly downloaded Paper jar.
-4. Start your new server.
+3. Rename the downloaded file to match the name specified in the [start command](../getting-started/getting-started.mdx#running-the-server)
```
Comment (zlataovce) [inline docs/paper/admin/getting-started/migration.md:25]:
```suggestion
3. Rename the downloaded file to match the name specified in the [start command](../getting-started/getting-started.mdx#running-the-server).
```

File: docs/paper/admin/getting-started/migration.md [hunk 2]
```diff
@@ -48,6 +49,7 @@ closely, as manual changes will be required.
 1. Stop your Vanilla server if it is running, and create a full backup.
 2. Download Paper from [our downloads page](https://papermc.io/downloads) and replace your Vanilla
    server jar with your freshly downloaded Paper jar.
+3. Rename the downloaded file to match the name specified in the [start command](../getting-started/getting-started.mdx#running-the-server)
```
Comment (zlataovce) [inline docs/paper/admin/getting-started/migration.md:52]:
```suggestion
3. Rename the downloaded file to match the name specified in the [start command](../getting-started/getting-started.mdx#running-the-server).
```

**Comment by PetarMc1** [comment]
@olijeffers0n Is it ok like this

File: docs/paper/admin/getting-started/migration.md [hunk 3]
```diff
@@ -22,17 +22,18 @@ It's easy to migrate from CraftBukkit or Spigot to Paper. Follow the steps below
 
 1. Stop your server if it is running, and create a full backup.
 2. Download Paper from [our downloads page](https://papermc.io/downloads).
-3. Replace your existing server jar with your freshly downloaded Paper jar.
-4. Start your new server.
+3. Rename the downloaded file to match the name specified in the [start command](../getting-started/getting-started.mdx#running-the-server).
```
Comment (zlataovce) [inline docs/paper/admin/getting-started/migration.md:25]:
```suggestion
3. Rename the downloaded file to match the name specified in the [start command](getting-started.mdx#running-the-server).
```

File: docs/paper/admin/getting-started/migration.md [hunk 4]
```diff
@@ -48,7 +49,8 @@ closely, as manual changes will be required.
 1. Stop your Vanilla server if it is running, and create a full backup.
 2. Download Paper from [our downloads page](https://papermc.io/downloads) and replace your Vanilla
    server jar with your freshly downloaded Paper jar.
-3. Start your new Paper server.
+3. Rename the downloaded file to match the name specified in the [start command](../getting-started/getting-started.mdx#running-the-server).
```
Comment (zlataovce) [inline docs/paper/admin/getting-started/migration.md:52]:
```suggestion
3. Rename the downloaded file to match the name specified in the [start command](getting-started.mdx#running-the-server).
```

**Review by PetarMc1 — COMMENTED** [review]
Thanks to @zlataovce  and @olijeffers0n  for suggesting changes

**Review by olijeffers0n — APPROVED** [review]

**Review by zlataovce — APPROVED** [review]

**Pull request merged**